framework/scripts/wazuh_server.py

```diff
@@ -348,24 +401,54 @@ def main():
 
         asyncio.run(main_function(args, cluster_configuration, cluster_items, main_logger))
     except KeyboardInterrupt:
-        main_logger.info("SIGINT received. Shutting down...")
+        main_logger.info('SIGINT received. Shutting down...')
     except MemoryError:
-        main_logger.error("Directory '/tmp' needs read, write & execution "
-                          "permission for 'wazuh' user")
+        main_logger.error("Directory '/tmp' needs read, write & execution " "permission for 'wazuh' user")
     except Exception as e:
-        main_logger.error(f"Unhandled exception: {e}")
+        main_logger.error(f'Unhandled exception: {e}')
     finally:
         shutdown_cluster(cluster_pid)
 
 
+def stop():
+    """Stop the Wazuh server running in background."""
+
+    pids_path = Path(common.OSSEC_PIDFILE_PATH)
+    try:
+        server_pid = next(pids_path.glob(f'{SERVER_DAEMON_NAME}-*.pid')).stem
+    except StopIteration:
+        main_logger.error('Wazuh server is not running.')
+        sys.exit(1)
+
+    server_pid = int(server_pid.split('-')[-1])
+    shutdown_cluster(server_pid)
+    os.kill(server_pid, signal.SIGKILL)
+
+
+def status():
+    """Show the status of the Wazuh server."""
+
+    daemons = [SERVER_DAEMON_NAME, COMMS_API_DAEMON_NAME, MANAGEMENT_API_DAEMON_NAME, ENGINE_DAEMON_NAME]
+    pids_path = Path(common.OSSEC_PIDFILE_PATH)
+    running_processes = [i.stem for i in pids_path.glob('wazuh-*-*') if '_' not in i.stem]
+    running_processes = ['-'.join(p.split('-')[:-1]) for p in running_processes]
```

Comment:
We could move this and the one in `stop()` to the pyDaemonModule.py file to keep everything related to processes/daemons there and to be able to re-use the functionality

Reply:
Done in b4b23f63ad4ba519afd723567a351888661a94b5